src/main/java/com/josketres/rfcfacil/JuristicPersonTenDigitsCodeCalculator.java

```diff
@@ -13,16 +13,17 @@
  */
 public class JuristicPersonTenDigitsCodeCalculator {
 
-    public static final String JURISTIC_PERSON_TYPE = " (S\\.? EN N\\.?C\\.?|" +
-            "S\\.? EN C\\.? POR A\\.?|" +
-            "S\\.? EN C\\.?|" +
-            "S\\.? DE R\\.?L\\.?|" +
-            "S\\.?A\\.? DE C\\.?V\\.?|" +
-            "A\\.? EN P\\.?|" +
-            "S\\.?C\\.?[LS]\\.?|" +
-            "S\\.?[AC]\\.?|" +
-            "S\\.?N\\.?C\\.?|" +
-            "A\\.?C\\.?)$";
+    public static final String JURISTIC_PERSON_TYPE = " (S\\.?\\s?EN\\s?N\\.?\\s?C\\.?|" +
+            "S\\.?\\s?EN\\s?C\\.?\\s?POR\\s?A\\.?|" +
```

Comment:
Claro... tengo varios ejemplos con los datos que he estado validando...

De hecho, el problema lo vi justo porque en la relación de razones sociales
con las que he probado, hay algunas en las que colocan (por error, o no)
espacios en las  abreviaturas de "S. A. de C. V." y otras...

En una oportunidad agrego los ejemplos...

Saludos.

ISC. Mario Jiménez Carrasco.
Subdirección Técnico y de Negocios
Desarrollo de Software.
Gobierna SCP.

2016-07-03 3:04 GMT-05:00 Josue Zarzosa <notifications@github.com>:

> In
> src/main/java/com/josketres/rfcfacil/JuristicPersonTenDigitsCodeCalculator.java
> <https://github.com/josketres/rfc-facil/pull/7#discussion_r69387334>:
>
> > @@ -13,16 +13,17 @@
> >   */
> >  public class JuristicPersonTenDigitsCodeCalculator {
> >
> > -    public static final String JURISTIC_PERSON_TYPE = " (S\\.? EN N\\.?C\\.?|" +
> > -            "S\\.? EN C\\.? POR A\\.?|" +
> > -            "S\\.? EN C\\.?|" +
> > -            "S\\.? DE R\\.?L\\.?|" +
> > -            "S\\.?A\\.? DE C\\.?V\\.?|" +
> > -            "A\\.? EN P\\.?|" +
> > -            "S\\.?C\\.?[LS]\\.?|" +
> > -            "S\\.?[AC]\\.?|" +
> > -            "S\\.?N\\.?C\\.?|" +
> > -            "A\\.?C\\.?)$";
> > +    public static final String JURISTIC_PERSON_TYPE = " (S\\.?\\s?EN\\s?N\\.?\\s?C\\.?|" +
> > +            "S\\.?\\s?EN\\s?C\\.?\\s?POR\\s?A\\.?|" +
>
> ¿Podrías incluir ejemplos para estos cambios en las expresiones regulares?
> Sería bueno que los agregaras como test unitario en
> JuristicPersonTenDigitsCodeCalculatorTest
> <https://github.com/josketres/rfc-facil/blob/master/src/test/java/com/josketres/rfcfacil/JuristicPersonTenDigitsCodeCalculatorTest.java>
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/josketres/rfc-facil/pull/7/files/a9d20a086440e8b43ac7f0bf06e70cbef3580325#r69387334>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AE4qFWzDVWSIbifUH3moBFbGyWI3MJ6Oks5qR20lgaJpZM4JDXvs>
> .
>